src/VirtoCommerce.CatalogModule.Web/Scripts/blades/editorialReview-detail.js

```diff
@@ -102,9 +103,10 @@ angular.module('virtoCommerce.catalogModule')
             var existReview = _.find(blade.item.reviews, function (x) { return x === blade.origEntity; });
             if (!existReview) {
                 blade.item.reviews.push(blade.origEntity);
-            };
+            }
 
             angular.copy(blade.currentEntity, blade.origEntity);
+            blade.currentEntity.isInherited = false;
```

Comment:
This flag should be changed before `angular.copy()`. Now it doesn't have any effect.